packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -914,7 +919,7 @@ function createIdealGraph(
         continue;
       }
       let reuseableBundleId = bundles.get(asset.id);
-      if (reuseableBundleId != null) {
+      if (reuseableBundleId != null && config.disableSharedBundles === false) {
```

Comment:
Could we maybe just wrap the whole section (the whole for loop) in a conditional checking the config, instead of checking it twice ?